code/__HELPERS/game.dm

```diff
@@ -31,6 +32,9 @@
 			return ERROR_BLOCKER
 	if(!target.check_alien_construction(null, TRUE))
 		return ERROR_CONSTRUCT
+	if(SSmonitor.gamestate == SHUTTERS_CLOSED)
+		if(target.get_build_prereqs())
+			return ERROR_INVALID_AREA
```

Comment:
Won't that still call get_build_prereqs the entire game though when we evaluate the condition?

Reply:
it evaluates conditions one at a time. Because it's an and check, it stops as soon as one fails, where as an OR condition would stop as soon as one passes.